package.json

```diff
@@ -1,8 +1,47 @@
 {
+  "name": "web-gphoto2",
+  "version": "1.0.0",
+  "description": "WebAssembly implementation of gphoto2 and libusb to control DSLR cameras over USB on the Web",
+  "type": "module",
+  "main": "src/camera.js",
+  "types": "src/libapi.mjs.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChromeLabs/web-gphoto2"
+  },
+  "keywords": [
+    "gphoto2",
+    "libusb",
+    "webassembly"
+  ],
+  "author": {
+    "name": "Ingvar Stepanyan",
+    "email": "me@rreverser.com",
+    "url": "https://rreverser.com"
+  },
+  "contributors": [
+    {
+      "name": "Tjeerd Bakker",
+      "email": "tjeerd992@gmail.com",
+      "url": "https://icheered.com"
+    }
+  ],
+  "license": "LGPL-2.1-or-later",
```

Comment:
Hm why did you change the license string? Let's keep it I think.

Reply:
During testing I got a warning that LGPLv2 is not a valid license string. 
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#license
It should be one of the following:
https://spdx.org/licenses/

Reply:
Huh okay.